### PR TITLE
Gidk 21a7a4 variant addone

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.html [hunk 1]
```diff
@@ -357,6 +357,194 @@ <h4>Product Dummy Name Furniture</h4>
...
+                                                                &nbsp;
+                                                            </label>
+                                                            <a href="#" class="font-22 d-flex"> + </a>
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                            </div>
+                                        </div>
+                                        <div class="row flex-column mr-t15">
+                                            <div class="col-xl-6 d-flex flex-column flex-lg-row  align-items-center expanse-form" *ngIf="!isExpense">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.html:527]: @shalinee-walkover correct expense spelling everywhere

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html [hunk 2]
```diff
@@ -965,6 +1167,216 @@
...
+                                            <label>
+                                                Item
+                                            </label>
+                                            <input type="text" id="service-addon-item" class="form-control" placeholder="Add item">
+                                        </div>
+                                        <div class="col-xl-2 col-md-6">
+                                            <label>
+                                                Quantity
+                                            </label>
+                                            <input type="text" id="service-addone-qty" class="form-control" placeholder="Enter Quantity">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:1228]: @shalinee-walkover correct addon spelling

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html [hunk 3]
```diff
@@ -1483,6 +1895,226 @@
...
+                                                Opening Balance
+                                            </label>
+                                            <input type="text" id="combo-variant-opening-balance" class="form-control"
+                                                placeholder="Enter amount">
+                                        </div>
+                                        <div class="col-xl-2 col-md-6">
+                                            <label>
+                                                Opening Qty
+                                            </label>
+                                            <input type="text" id="combo-vairant-qty" class="form-control" placeholder="Enter Qty">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:1928]: @shalinee-walkover correct variant spelling

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html [hunk 4]
```diff
@@ -1483,6 +1895,226 @@
...
+                                        </div>
+                                        <div class="col-xl-2 col-md-6">
+                                            <label>
+                                                Rate
+                                            </label>
+                                            <input type="text" id="combo-vairant-rate" class="form-control" placeholder="Enter Rate">
```
Comment (aditi-tiwari17) [inline apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html:1934]: @shalinee-walkover correct variant spelling

**Review by ravinderwalkover — APPROVED** [review]

File: apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.html (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/about-product-service-detail/about-product-service-detail.component.html]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html]: Done

File: apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/new-inventory/component/create-new-inventory-component/create-new-inventory.component.html]: Done

**Review by aditi-tiwari17 — APPROVED** [review]

**Pull request merged**